tests/test_importer_utils.py

```diff
@@ -79,6 +79,16 @@ def test_get_text_inside_brackets_succeed(self):
         output = "Kulla Gunnarstorp 1:21"
         self.assertEqual(utils.get_text_inside_brackets(text), output)
 
+    def test_get_text_inside_brackets_nested_one(self):
+        text = "outsidetext (foo (cat))"
+        output = "foo (cat)"
+        self.assertEqual(utils.get_text_inside_brackets(text), output)
+
+    def test_get_text_inside_brackets_nested_two(self):
+        text = "text (foo (bar (cat))) around"
+        output = "foo (bar (cat))"
+        self.assertEqual(utils.get_text_inside_brackets(text), output)
```

Comment:
Add a test for `pre (foo) mid (bar) post`.
Expected behaviour (in my opinion) is one of the following:

1. Return only `foo`
2. Return `(foo, bar)` (or `[foo, bar]`)
3. Raise an error

What I believe will happen right now is `foo) mid (bar`